pyoos/parsers/nerrs.py

```diff
@@ -116,7 +119,7 @@ def get_station(feature):
             s = Station()
             s.uid = feature['Station_Code']
             s.name = feature['Station_Name']
-            s.location = sPoint(float(feature['Longitude']), float(feature['Latitude']), 0)
+            s.location = sPoint(feature['Longitude'], feature['Latitude'], 0)
```

Comment:
No need to make them `float` here if `feature` was already parsed above.  Thanks!